draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,16 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint SHOULD limit the state it commits to retiring connection IDs. An
```

Comment:
I feel like "commits to" is unclear. 

Perhaps
"An endpoint should limit the number of connection IDs that it has retired locally and have not yet been acknowledged."